compute_sdk/globus_compute_sdk/sdk/login_manager/authorizer_login_manager.py

```diff
@@ -22,6 +23,13 @@ class AuthorizerLoginManager(LoginManagerProtocol):
     """
 
     def __init__(self, authorizers: dict[str, globus_sdk.RefreshTokenAuthorizer]):
+        warnings.warn(
+            "The `AuthorizerLoginManager` is deprecated. Please use `GlobusApp` objects"
+            "from the Globus SDK instead:"
```

Comment:
" ... objectsfrom ... " (missing prefix space on second line)

Reply:
Looks like copy-pasta included this in a couple of spots.

Reply:
Good catch.